### PR TITLE
Fix Testworld NFT Test

### PR DESCRIPTION
**Pull request by vimukthi-git**
<This is intended as a guide when describing-reviewing pull requests for go-centrifuge>
Issue Link: <Github link>

### CODE REVIEW

File: testworld/nft_test.go [hunk 1]
```diff
@@ -130,10 +132,14 @@ func paymentObligationMint(t *testing.T, documentType string, grantNFTAccess, to
...
-
-	assert.Nil(t, err, "mintNFT should be successful")
+	assert.NoError(t, err, "mintNFT should be successful")
 	assert.True(t, len(response.Value("token_id").String().Raw()) > 0, "successful tokenId should have length 77")
 
+	tokenID, err := nft.TokenIDFromString(response.Value("token_id").String().Raw())
+	assert.NoError(t, err, "token ID should be correct")
+	owner, err := alice.host.tokenRegistry.OwnerOf(registry, tokenID.BigInt().Bytes())
+	assert.NoError(t, err)
+	assert.Equal(t, strings.ToLower("0x44a0579754d6c94e7bb2c26bfa7394311cc50ccb"), strings.ToLower(owner.Hex()))
```
Comment (mikiquantum) [inline testworld/nft_test.go]: I think you meant to pass here the deposit address of the test context.

File: documents/invoice/model.go [hunk 2]
```diff
@@ -487,18 +486,3 @@ func (i *Invoice) CollaboratorCanUpdate(updated documents.Model, collaborator id
 	cf := documents.GetChangedFields(oldTree, newTree, proofs.DefaultSaltsLengthSuffix)
 	return documents.ValidateTransitions(rules, cf)
 }
-
```
Comment (mikiquantum) [inline documents/invoice/model.go]: Are these defined somewhere else? or are not needed.

File: anchors/service.go [hunk 3]
```diff
@@ -54,7 +54,13 @@ func (s *service) GetAnchorData(anchorID AnchorID) (docRoot DocumentRoot, anchor
 	// Ignoring cancelFunc as code will block until response or timeout is triggered
 	opts, _ := s.client.GetGethCallOpts(false)
 	r, err := s.anchorRepositoryContract.GetAnchorById(opts, anchorID.BigInt())
+	if err != nil {
```
Comment (mikiquantum) [inline anchors/service.go]: can r be nil and not error thrown?

File: testworld/nft_test.go (hunk 1 above)
Comment (vimukthi-git) [reply testworld/nft_test.go]: we always deposit to our address to make to debug easily

File: documents/invoice/model.go (hunk 2 above)
Comment (vimukthi-git) [reply documents/invoice/model.go]: they are not needed as @vedhavyas pointed out to me today

File: testworld/nft_test.go (hunk 1 above)
Comment (mikiquantum) [reply testworld/nft_test.go]: even against local envs?

File: anchors/service.go (hunk 3 above)
Comment (vimukthi-git) [reply anchors/service.go]: let me add a check for those

Comment (vimukthi-git) [reply anchors/service.go]: r cant be nil but blk can so added a check

File: testworld/nft_test.go (hunk 1 above)
Comment (vimukthi-git) [reply testworld/nft_test.go]: yes, its the same hardcoded address for all

File: anchors/service.go [hunk 4]
```diff
@@ -5,6 +5,8 @@ import (
 	"math/big"
 	"time"
 
+	"github.com/centrifuge/go-centrifuge/errors"
+
```
Comment (vedhavyas) [inline anchors/service.go]: sort imports pls

**Pull request merged**